keyboards/tada68/keymaps/mattgemmell/keymap.c

```diff
@@ -6,43 +6,43 @@
+    KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_NUBS, \
     KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,          KC_DEL,  \
     KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_NUHS, KC_ENT,  KC_PGUP, \
-    KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_PGDN, \
+    KC_LSFT, KC_GRV,  KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_PGDN, \
     KC_LCTL, KC_LALT, KC_LGUI,                   KC_SPC,                             KC_RALT, MO(_FL), KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
 	),
 
   /* Keymap _FL1: Function Layer 1
    * ,----------------------------------------------------------------.
-   * |   | F1| F2| F3| F4| F5| F6| F7| F8| F9|F10|F11|F12|       | :D |
+   * |   | F1| F2| F3| F4| F5| F6| F7| F8| F9|F10|F11|F12|       |BR↑ |
    * |----------------------------------------------------------------|
-   * |     |LMB| Up|RMB|   |   |   |   |   |   |   |   |   |     |    |
+   * |     |   |   |   |   |   |   |   |   |   |   |   |   |     |BR↓ |
    * |-------------------------------------------------------    -----|
-   * |       |Lef|Dow|Rig|   |   |   |   |   |   |   |  |   |    |Home|
+   * |       |   |   |   |   |   |   |   |   |   |   |  |   |    |Home|
    * |----------------------------------------------------------------|
-   * |     |   |   |   | L+|LED| L-|   | V+|  V-|Mut|  | MsBtn|↑ | End|
+   * |     |   |   |   | L+|LED| L-|   |   | V↓| V↑|Mut|     | ↑ | End|
    * |----------------------------------------------------------------|
-   * |    |    |    |      Reset            |   |   |    | ← | ↓ | →  |
+   * |    |    |    |                       |   |   |    | ← | ↓ | →  |
    * `----------------------------------------------------------------'
    */
   [_FL] = LAYOUT_iso(
-    _______, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, _______, \
-    _______, KC_BTN1, KC_UP,   KC_BTN2, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
-    _______, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______, _______, _______, _______, _______, _______, _______,          _______, KC_HOME, \
-    _______, _______, _______, _______, BL_DEC,  BL_TOGG, BL_INC,  _______, KC_VOLU, KC_VOLD, KC_MUTE, _______, _______, KC_MS_U, KC_END,  \
+    _______, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, KC_BRMU, \
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_BRMD, \
```

Comment:
You've got these in the wrong order on this layer, which will lead to keycodes being on the wrong keys. Apply this suggestion and the next to fix it.

```suggestion
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          KC_BRMD, \
```